Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -811,6 +914,124 @@ def ip_list_get_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> Co
     )
 
 
+def ip_lists_get_command(
+    client: PolicyComputeEngine, args: Dict[str, Any]
+) -> CommandResults:
+    """Retrieve the IP lists.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        CommandResult object
+    """
+    description = args.get("description")
+    fqdn = args.get("fqdn")
+    ip_address = args.get("ip_address")
+    max_results = arg_to_number(
+        args.get("max_results", "500"), arg_name="max_results"
+    )
+    name = args.get("name")
+
+    validate_ip_lists_get_arguments(max_results)
+
+    ip_lists = client.ip_lists.get_all(
```

Comment:
Don't use `get_all` here - this function is meant to be used to pull everything regardless of `max_results` (currently if `max_results` is provided it just performs a normal GET, but this may change in the future).

Reply:
will instead use `get` call for `ip_lists`